cfgov/unprocessed/js/modules/util/web-storage-proxy.js

```diff
@@ -81,6 +81,7 @@ function setStorage( storage ) {
   _storage = storage;
 }
 
+/* istanbul ignore next */
```

Comment:
I'll take a look at the istanbul report to make sure it's still ignoring properly.